wbooks/src/app/screens/dashboard/screens/Home/components/SearchBar/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.css';
+import searchSvg from './assets/search.svg'
+
+class SearchBar extends React.Component {
+  handleSubmit = event => {
+    event.preventDefault();
+    const value = this.textInput.value;
+    this.props.onSubmit(value);
```

Comment:
`onSubmit` is missing props validation. It is a `PropTypes.func`. Should it be `required`?